server/api/api.go

```diff
@@ -2966,6 +2967,85 @@ func (a *API) handleDuplicateBlock(w http.ResponseWriter, r *http.Request) {
 	auditRec.Success()
 }
 
+func (a *API) handleGetBoardMetadata(w http.ResponseWriter, r *http.Request) {
```

Comment:
This API is missing the check for Mattermost enterprise edition.

Reply:
I've implemented the license check - by exposing GetLicence on the Store, and overriding this in mattermostauthlayer. Also added integration tests to cover this.